align_system/configs/adm/outlines_regression_aligned_comparative/incontext.yaml

```diff
@@ -7,11 +7,8 @@ inference_kwargs:
     number: 5
     method: prompt_bert_similarity
     leave_one_out: false
-    normalization: globalnorm
+    normalization: null
+    sort_actions: true
     datasets:
-      MoralDesert: /data/shared/samba/integrated_results_metrics_eval/captured_dataset_for_chris/baseline_adept_high-1715105775-input-output.json
-      maximization: /data/shared/samba/integrated_results_metrics_eval/captured_dataset_for_chris/baseline_soartech_high-1716581856-input-output.json
-      Moral judgement: /data/shared/samba/dry_run/moral_judgement_20240826.json
-      Ingroup Bias: /data/shared/samba/dry_run/ingroup_bias_20240826.json
-      QualityOfLife: /data/shared/samba/dry_run/qol_20240826.json
-      PerceivedQuantityOfLivesSaved: /data/shared/samba/dry_run/vol_20240826.json
+      QualityOfLife: /data/shared/samba/phase1_icl/qol_train1_20241105.json
+      PerceivedQuantityOfLivesSaved: /data/shared/samba/phase1_icl/vol_train1_20241105.json
```

Comment:
Wondering why we're removing the other KDMAs here, are we only using incontext for the SoarTech KDMAs now?  (Maybe I'm missing some context for these changes)

Reply:
We haven't created the updated Phase1 ICL files for Adept yet. I took them out for now so that we didn't have a mix of old (DRE) and new ICL jsons in there

Reply:
These SoarTech files will also eventually be replaced with ones created from the final versions of all three train files

Reply:
So I think the problem I'm still having then is that if we merge this in as-is, we won't be able to smoketest ICL on adept; I feel like a smoother transition would be to just update the ICL databases that are updated.  Would a feasible alternative here be to create a new config file here called `incontext_phase1.yaml` or something like that, which can be more volatile leading up the the eval, but the `incontext.yaml` is the stable older version used for DRE (or maybe `incontext_dre.yaml`?

Reply:
Makes sense, I'll change it to `incontext_phase1.yaml` and keep the old one

Reply:
Done

Reply:
Thanks Jadie, merging shortly